packages/interactivity/src/vdom.ts

```diff
@@ -127,18 +131,22 @@ export function toVdom( root ) {
 							SCRIPT_DEBUG === true
 						) {
 							// eslint-disable-next-line no-console
-							console.warn( `Invalid directive: ${ name }.` );
+							console.warn(
+								`Invalid directive: ${ directiveName }.`
+							);
 						}
 						return obj;
 					}
-					const prefix = directiveMatch[ 1 ] || '';
-					const suffix = directiveMatch[ 2 ] || 'default';
-
-					obj[ prefix ] = obj[ prefix ] || [];
-					obj[ prefix ].push( {
-						namespace: ns ?? currentNamespace(),
-						value,
-						suffix,
+					// Splitting the directive name into directive type and input.
+					const directiveType = directiveMatch[ 1 ] || '';
+					const directiveInput = directiveMatch[ 2 ] || 'default';
```

Comment:
Could be "event" or "directiveEvent" a better name for this?

Not sure, cause they are events in `data-wp-on-document` or `data-wp-on-window` or `data-wp-on`. But can also be styles or classes in `data-wp-class--` or `data-wp--style`.

What do you think about it? Let's ping some folks with experience developing with the Interactivity API to get some feedback.

Ping: @ryanwelcher , @luisherranz , @DAreRodz or @sethrubenstein